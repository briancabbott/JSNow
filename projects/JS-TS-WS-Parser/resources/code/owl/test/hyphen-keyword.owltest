a->b
---
rule = identifier : ident
 .operators infix left '->' : arrow
