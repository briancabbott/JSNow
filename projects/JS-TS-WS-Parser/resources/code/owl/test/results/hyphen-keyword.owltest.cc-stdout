rule : ARROW (0 - 4)
  rule@left : IDENT (0 - 1)
    identifier - a (0 - 1)
  rule@right : IDENT (3 - 4)
    identifier - b (3 - 4)
