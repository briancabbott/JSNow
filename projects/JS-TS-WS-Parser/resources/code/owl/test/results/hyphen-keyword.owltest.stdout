. a         ->b         
  rule:ident  rule:ident
  rule:arrow------------
